CPP17.md

```diff
@@ -237,6 +238,26 @@ byte d = byte{1}; // OK
 byte e = byte{256}; // ERROR
 ```
 
+### Standard Attributes
+C++17 introduces three new attributes: `[[fallthrough]]`, `[[nodiscard]]` and `[[maybe_unused]]`:
+```c++
+// Will warn if return of foo() is ignored
+[[nodiscard]] int foo();
+int main () {
+  int a {1};
+  switch(a)
+  {
```

Comment:
Put the brace on the same line as the switch. Also, add a space between `switch` and the left parenthesis.